.coveragerc

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2021 The Atlite Authors
+# SPDX-FileCopyrightText: Contributors to Atlite <https://github.com/pypsa/atlite>
```

Comment:
```suggestion
# SPDX-FileCopyrightText: Contributors to atlite <https://github.com/pypsa/atlite>
```

`atlite` is written with a small starting letter, since it is a name that pattern should be preserved here and elsewhere

Reply:
Great, I changed it also everywhere else (docs, Readme etc.)

Reply:
Thanks!